### PR TITLE
[Fixed #2570] Get code completion for LibraryComponent.library

### LINKED ISSUE #2570: [Bug] Get code completion for LibraryComponent.library does not work correctly
this is just a notice from RoboCon

If I rename__init__.pyi to browser.pyi it works for VSCode.

### PR DESCRIPTION
**Pull request by Snooz82**
Signed-off-by: René <snooz@posteo.de>

### CODE REVIEW

File: Browser/entry.py [hunk 1]
```diff
@@ -206,11 +206,11 @@ def show_versions():
     version_file = CURRENT_FOLDER / "version.py"
     version_text = version_file.read_text()
     match = re.search(r"\"\d+\.\d+.\d+\"", version_text)
-    browser_lib_version = match.group(0)
+    browser_lib_version = match.group(0) if match else "unknown"
     package_json = INSTALLATION_DIR / "package.json"
     package_json_data = json.loads(package_json.read_text())
     match = re.search(r"\d+\.\d+\.\d+", package_json_data["dependencies"]["playwright"])
-    pw_version = match.group(0)
+    pw_version = match.group(0) if match else "unknown"
```
Comment (aaltat) [inline Browser/entry.py:213]: I would just raise an exception, it gets noticed better in that case

File: Browser/entry.py [hunk 2]
```diff
@@ -206,11 +206,11 @@ def show_versions():
     version_file = CURRENT_FOLDER / "version.py"
     version_text = version_file.read_text()
     match = re.search(r"\"\d+\.\d+.\d+\"", version_text)
-    browser_lib_version = match.group(0)
+    browser_lib_version = match.group(0) if match else "unknown"
```
Comment (aaltat) [inline Browser/entry.py:209]: I would just raise an exception, it gets noticed better in that cas

File: atest/test/09_Plugins/plugin.robot [hunk 3]
```diff
@@ -55,7 +55,7 @@ Pluging Keyword Example Location
     Should Be Equal    ${location.protocol}    http:
     Should Be Equal    ${location.href}    ${FORM_URL}
 
-# TODO  ADD THESE TESTS WHEN NEW PLC IS RELEASED
+# TODO    ADD THESE TESTS WHEN NEW PLC IS RELEASED
```
Comment (aaltat) [inline atest/test/09_Plugins/plugin.robot:58]: Is the comment correct and test could be returned?

File: Browser/entry.py (hunk 1 above)
Comment (Snooz82) [reply Browser/entry.py:213]: yea but it is just an information to the user.
Therefore the user would get scared by that exception.

**Pull request merged**